menas/src/main/scala/za/co/absa/enceladus/menas/models/rest/errors/RequestTimeoutExpiredError.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.menas.models.rest.errors
+
+import za.co.absa.enceladus.menas.models.rest.ResponseError
+
+/**
+ * This error is produced when a request timeout expires.
+ */
+case class RequestTimeoutExpiredError(errorType: String = "request_timeout_expired") extends ResponseError
```

Comment:
Why the default message here is lower cases with underscores and in `RestExceptionHandler.scala` line 46 actually proper sentence is used?

Reply:
It's `errorType`, so that clients (e.g. frontend) can do switch/case on it. A message is constructed separately as part of the response.

The idea behind this is making errors as strongly typed as possible so that clients can reuse the context. This emerged from schema parsing errors so that frontend can point at the exact position in a schema file where it is failed to parse.

Reply:
Eh, sorry. It's `errorType` not `message`, misread.